serve/mlc_serve/model/tvm_model.py

```diff
@@ -140,6 +139,17 @@ def __init__(
         self.sliding_window = config.sliding_window
         self.num_shards = config.num_shards
 
+        # TODO(@sunggg): Find a better way
```

Comment:
@sunggg Can we potentially refactor all of the config to also store this information in an `enum` or something?

Reply:
Create a ticket for this as well.